src/main/kotlin/com/openlattice/shuttle/pods/ShuttleServicesPod.kt

```diff
@@ -444,6 +437,12 @@ class ShuttleServicesPod {
         )
     }
 
+    @Bean
+    fun elasticsearchApi(): ConductorElasticsearchApi {
+        val localConductorConfiguration = getLocalConductorConfiguration()
```

Comment:
This will always load the local conductor configuration. Not sure if this will be compatible with rundeck.

Reply:
Right. This is what I was hoping you'd comment on. I want to do like ... `getBean("conductorConfiguration")` ... since both beans have that `name`, but I don't know how. I know `getBean` is a thing but I don't know how to get access to it inside the pod.

Reply:
You have to break into a separate pod that exposes the beans. AuditConfigurationPod might be a relevant example. Then just inject conductorconfiguration.